SECP-service/src/main/resources/assets/app/mockedResponses/chatList.json

```diff
@@ -0,0 +1,61 @@
+[
+    {
+        "id" : "1",
```

Comment:
His is nice to test with untill we have all the dB stuff, works as expected

Reply:
True :)